plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlPlugin.java

```diff
@@ -15,11 +15,13 @@
 
 import io.trino.plugin.jdbc.JdbcPlugin;
 
+import static io.airlift.configuration.ConfigurationAwareModule.combine;
+
 public class PostgreSqlPlugin
         extends JdbcPlugin
 {
     public PostgreSqlPlugin()
     {
-        super("postgresql", new PostgreSqlClientModule());
+        super("postgresql", combine(new PostgreSqlClientModule(), new PostgreSqlClientAuthenticationModule()));
```

Comment:
Would it make sense to extract this refactor to a prep commit?

It would then require some explanation why we're doing this, but would also make seeing the refactor as a whole simpler.